.github/dependabot.yml

```diff
@@ -4,3 +4,7 @@ updates:
     directory: "/"
     schedule:
       interval: "weekly"
+  - package-ecosystem: "gitsubmodule"
+    directory: "/tests/"
+    schedule:
+      interval: "weekly"
```

Comment:
We don't want dependabot to push updates for libmonc and libmongocrypt.

Reply:
Would it make sense to specify the path to the actual submodule, or does this need to be a level above?

Reply:
I'll try with the submodule path; it should work.